ui-tests/cypress/integration/content/assets.spec.js

```diff
@@ -122,69 +130,104 @@ describe('Assets', () => {
     describe([Tag.GTS], 'Edit asset', () => {
 
       it('Edit description', () => {
-        currentPage = openAssetsPage();
-        //TODO validate this wait
-        cy.wait(2000);
-        cy.get('@assetToBeDeleted')
-          .then(assetToBeDeleted => currentPage.getContent().getKebabMenu(assetToBeDeleted.id).open().openEdit())
-          .then(() => {
-            currentPage.getDialog().getBody().getDescriptionInput().type('test');
-            currentPage.getDialog().confirm();
-          });
-
-        cy.validateToast(currentPage, 'test');
+        openAssetsPage()
+            .then(page => {
+              cy.get('@assetToBeDeleted')
+                .then(() => {
+                  page.getContent().getKebabMenu().openDropdown().openEdit();
+                })
+                .then(page => {
+                  page.getContent().getDescriptionInput().clear().type('test');
+                  page.getContent().submit();
+                })
+                .then(page => {
+                  page.getContent().getTableRows().then(rows =>
+                      cy.wrap(rows).children(htmlElements.div).should('contain.text', 'test'));
+                });
+            });
       });
 
       it('Crop image', () => {
-        currentPage = openAssetsPage();
-        cy.wait(2000);
-        cy.get('@assetToBeDeleted')
-          .then(assetToBeDeleted => currentPage.getContent().getKebabMenu(assetToBeDeleted.id).open().openEdit())
-          .then(() => {
-            currentPage.getDialog().getBody().crop(-100, -100);
-            currentPage.getDialog().confirm();
-          });
+        openAssetsPage()
+            .then(page => {
+              cy.get('@assetToBeDeleted')
```

Comment:
Why do you get `@assetToBeDeleted` if you're not using it?

Reply:
I have removed all `@assetToBeDeleted` ones, I have no idea why it is still here :(

Reply:
I have removed all `@assetToBeDeleted` ones, I have no idea why it is still here :(